client-v2/src/components/FrontsEdit/ArticleFragmentForm.js

```diff
@@ -0,0 +1,354 @@
+// @flow
+
```

Comment:
As mentioned in the PR description, this form file is large; we could look at splitting some of the transform in/out logic out of it.